src/shared/tests.h

```diff
@@ -332,6 +314,9 @@ static inline int run_test_table(void) {
                 }                                                                                               \
         })
 
+#define ASSERT_TRUE(expr)  ASSERT_EQ((int) (expr), 1)
+#define ASSERT_FALSE(expr) ASSERT_EQ((int) (expr), 0)
```

Comment:
I'm not a big fan of this because the error will not be very informative if the function call with negative errno fails. I'd prefer we add `ASSERT_OK_POSITIVE` and `ASSERT_OK_ZERO` instead.